src/app/organization/organization-routing.module.ts

```diff
@@ -164,7 +165,8 @@ const routes: Routes = [
               path: '',
               component: OfficesComponent,
               resolve: {
-              offices: OfficesResolver
+              offices: OfficesResolver,
+              officeDataTables: OfficeDataTablesResolver
```

Comment:
```suggestion
              officeDataTables: OfficeDatatablesResolver
```
Resolver already there in common resolvers